### PR TITLE
Health Data

### PR DESCRIPTION
**Pull request by szabozoltan69**
Refers to https://github.com/IFRCGo/go-api/issues/2084 and to https://github.com/IFRCGo/go-api/issues/2091

Design Link
- https://www.figma.com/proto/H9mZfGeXtlzK6HPzttYoci/IFRC-GO---Wires-Current---1?page-id=2%3A68604&type=design&node-id=3856-49130&viewport=2538%2C-1789%2C0.28&t=bOW8frnGS6h41erL-1&scaling=scale-down-width&starting-point-node-id=3856%3A49130&hide-ui=1

### CODE REVIEW

**Comment by szabozoltan69** [comment]
@thenav56 please give some hints about this change.
(The visibility (instead of is_public) is still not used, just "exist". Also no serializers and endpoints for the new health data. The foreign key is ok in LocalUnit table to HealthData table? Or should it be vica versa?)
The showing "coverage" instead of "level" was a request of @davidmuchatiza .

File: local_units/admin.py [hunk 1]
```diff
@@ -60,6 +68,66 @@ class DelegationOfficeAdmin(admin.OSMGeoAdmin):
...
 
+
 @admin.register(DelegationOfficeType)
 class DelegationOfficeTypeAdmin(admin.ModelAdmin):
-    ordering=('code',)
+    ordering = ('code',)
+
+
+@admin.register(Affiliation)
+class Affiliation(admin.ModelAdmin):
```
Comment (k9845) [inline local_units/admin.py:78]:
```suggestion
class AffiliationAmin(admin.ModelAdmin):
```

File: local_units/admin.py [hunk 2]
```diff
@@ -60,6 +68,66 @@ class DelegationOfficeAdmin(admin.OSMGeoAdmin):
...
+
+@admin.register(ProfessionalTrnFacility)
+class ProfessionalTrnFacility(admin.ModelAdmin):
+    ordering = ('code',)
+    search_fields = ('name',)
+
+
+@admin.register(HealthData)
+class HealthData(admin.ModelAdmin):
+    search_fields = ('affiliation',)
```
Comment (k9845) [inline local_units/admin.py:133]: Add prefix `Admin` for all class name.

File: local_units/models.py [hunk 3]
```diff
@@ -2,7 +2,256 @@
...
+    code = models.IntegerField(
+        validators=[MinValueValidator(0), MaxValueValidator(99)])
+    name = models.CharField(
+        max_length=100, verbose_name=_('Name'))
+
+    def __str__(self):
+        return f'{self.name} ({self.code})'
+
+
+class ProfessionalTrnFacility(models.Model):
```
Comment (k9845) [inline local_units/models.py:96]:
```suggestion
class ProfessionalTrainingFacility(models.Model):
```

File: local_units/models.py [hunk 4]
```diff
@@ -82,7 +335,9 @@ class LocalUnit(models.Model):
     )
     draft = models.BooleanField(default=False, verbose_name=_('Draft'))
     validated = models.BooleanField(default=False, verbose_name=_('Validated'))
-    is_public = models.BooleanField(default=False, verbose_name=_('Is public?'))
+    visibility = models.IntegerField(
+        choices=VisibilityChoices.choices, verbose_name=_('visibility'),
+        default=2)  # 2:IFRC
```
Comment (k9845) [inline local_units/models.py:340]: We also need to verify this with new data excel file.

**Review by k9845 — CHANGES_REQUESTED** [review]

**Comment by k9845** [comment]
@szabozoltan69 Can you also rebase this branch with `develop`?

File: local_units/models.py (hunk 4 above)
Comment (szabozoltan69) [reply local_units/models.py:340]: What do you mean on verify?
(And which one is the "new" data excel file?)

**Review by k9845 — APPROVED** [review]
LGTM

**Pull request merged**